modules/relay/src/main/RelayPlayerEnrich.scala

```diff
@@ -50,27 +50,18 @@ private case class RelayPlayersTextarea(text: String):
     lines.nonEmpty.so:
       text.linesIterator.take(1000).toList.flatMap(parse).toMap
 
-  // Original name / Optional rating / Optional title / Optional replacement name
+  // Original name / Optional FideID / Optional title / Optional rating / Optional replacement name
   private def parse(line: String): Option[(PlayerName, RelayPlayerLine)] =
```

Comment:
Wouldn't this change break all previous broadcasts replace?

Reply:
yes, it will break the previous Broadcasts a little...

but it also doesn't make sense to have 2 different systems (if 1 is used, it will alert broadcasters)

Reply:
Maybe we could do what happened with the change of the old system: leave "a month" with the system running 2 systems and then delete the old one